doc/s9s-cluster.1

```diff
@@ -1178,6 +1178,130 @@ cluster this option can be used to set tags for the newly created cluster.
 Install the TimescaleDB option when creating a new cluster. This is currently
 only supported on PostgreSQL systems.
 
+.TP
+.B \-\-add\-publication
+Creates a new PostgreSQL publication for logical replication cluster.
+
+.B EXAMPLE
+.nf
+s9s cluster \\
+    --add-publication \\
+    --cluster-id=1 \\
+    --subcluster-id=2 \\
+    --pub-name=my_publication \\
+    --db-name=mydb \\
+    --include-all-tables
+
+s9s cluster \\
+    --add-publication \\
+    --cluster-id=1 \\
+    --subcluster-name=source_db \\
+    --pub-name=sales_pub \\
+    --db-name=sales_db \\
+    --db-tables="orders,customers,products"
+.fi
+
+The operation requires either \fB\-\-include\-all\-tables\fP to include all tables
+in the publication or \fB\-\-db\-tables\fP to specify a list of tables.
+
+.TP
+.B --drop-publication
+Drops an existing PostgreSQL publication.
+
+.B EXAMPLE
+.nf
+s9s cluster \\
+    --drop-publication \\
+    --cluster-id=1 \\
+    --subcluster-id=2 \\
+    --pub-name=my_publication \\
+    --db-name=mydb
+.fi
+
+.TP
+.B --list-publications
+Lists all publications in a PostgreSQL Logical Replication cluster.
+
+.B EXAMPLE
+.nf
+s9s cluster \\
+    --list-publications \\
+    --cluster-id=1 \\
+    --subcluster-name=source_db
+.fi
+
+.TP
+.B --add-subscription
+Creates a new PostgreSQL subscription for logical replication.
+
+.B EXAMPLE
+.nf
+s9s cluster \\
+    --add-subscription \\
+    --cluster-id=1 \\
+    --subcluster-id=3 \\
+    --sub-name=my_subscription \\
```

Comment:
We might need some improvement here later. May be good to think about it now too.
We might have to support the case when subscription targets a replica node on the publication(source) cluster side. In those cases it is not enough to specify the subcluster id, but the exatc hostname and port have to be defined.
May be Later we can add support for --node here, and if --node is specified, we don't have to require the subcluster id.